# Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of
15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

pop mean = 100,
pop dev = 15,
sample mean = 108,
sample size = 36,
significance level = 5% (Assume),
H0 = mean >100,
H1 = mean <100,
It is a one tail test (left),
z(0.05)=-1.645


In [13]:
u =100
s =15
x = 108
n = 36
std_error = s/(n**0.5)


In [14]:
print(std_error)

2.5


In [15]:
z_test = (x-u)/std_error
print(z_test)

3.2


z_test > z(0.05),
p(3.2)=0.9993,
p(3.2)> 0.05 hence we cannot reject the null hypothesis

# Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

In [14]:
n1 =100
n2 =100
P1 = 0.52
P2 = 0.47

In [24]:
diff = (52-47)/100

In [25]:
diff

0.05

In [26]:
m1 = P1*(1-P1)/n1
m1

0.002496

In [27]:
m2 = P2*(1-P2)/n1
m2

0.0024909999999999997

In [28]:
std=(m1+m2)**0.5

In [29]:
std

0.07061869440877536

In [30]:
x =0

In [31]:
z =(x - diff)/std
z

-0.7080278164104207

p(-x)=1-p(x)

In [1]:
px = 1 -0.7823
px

0.2177

Therefore, the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.2177.

# Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

H0 (>1026), H1 (<1026), It is a one tail test (left), z(0.05)=-1.645

In [20]:
u =1026
s =209
x = 1100
n = 1
std_error = s/(n**0.5)

In [21]:
std_error

209.0

In [22]:
z_test = (x-u)/std_error
print(z_test)

0.35406698564593303


z_test > z(0.05),
p(0.35)=0.6368,
p(0.35)> 0.05 hence we cannot reject the null hypothesis